db/migrate/20191007140446_add_user_accuracy_bonus_to_reviewable_scores.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddUserAccuracyBonusToReviewableScores < ActiveRecord::Migration[6.0]
+  def up
+    # Add user_accuracy_bonus to reviewable_scores
+    add_column :reviewable_scores, :user_accuracy_bonus, :float, default: 0
```

Comment:
Very minor but I'd add a `null: false` here to make sure the column is not nullable.